test/e2e/util.go

```diff
@@ -333,7 +341,7 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	kubectlArgs := append(defaultArgs, args...)
 
 	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command("kubectl", kubectlArgs...)
+	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
```

Comment:
Looking at `cluster/kubectl.sh`, it looks like we can set the `KUBECTL_PATH` env variable for the gke/gke-test projects, and then it'll actually use the right version. So I'll keep the change to use cluster/kubectl.sh rather than kubectl directly? Unless there's still a strong reason not to.